scripts/generator/graphql_generator/csharp/SDK/iOS/iOSWebCheckout.cs.erb

```diff
@@ -0,0 +1,51 @@
+#if UNITY_IPHONE
+namespace <%= namespace %>.SDK.iOS {
+#if !SHOPIFY_MONO_UNIT_TEST
+    using UnityEngine;
+    using System.Runtime.InteropServices;
+    using <%= namespace %>.SDK;
+
+    class iOSWebCheckout: IWebCheckout {
+        private static iOSWebCheckoutBehaviour WebComponent;
```

Comment:
I feel that `iOSWebCheckoutBehaviour` isn't a really descriptive name. Really what does `iOSWebCheckoutBehaviour` do? It receives messages from the native module. Should we call `iOSWebCheckoutBehaviour` something like `iOSWebviewMessageReceiver`. I'm not sure the names should ever include `Behaviour` or `Component` because in Unity it's essentially noise because everything is either a `MonoBehaviour` or an instance / `Component` of a `MonoBehaviour`.